src/App.js

```diff
@@ -28,9 +28,11 @@ import { CONSTANTS } from './constant';
 require('dotenv').config({ override: true });
 import { checkMockOSRestInterface, TRUE_VALUES, getCurrentAppID } from './utils/Utils';
 import { AcknowledgeChallenge, Keyboard, PinChallenge } from '@firebolt-js/manage-sdk';
+import { Discovery } from '@firebolt-js/sdk';
 import PinChallengeProviderDelegater from './providers/PinChallengeDelegater';
 import KeyboardProviderDelegater from './providers/KeyboardProviderDelegater';
 import AckChallengeProviderDelegater from './providers/AckChallengeDelegater';
+import UserIntrestDelegater from './providers/UserInterestDelegater';
 const logger = require('./utils/Logger')('App.js');
```

Comment:
spelling is wrong for UserIntrestDelegater

Reply:
corrected